CMakeLists.txt

```diff
@@ -317,7 +317,11 @@ if(NOT MSVC71 AND NOT MSVC80)
     set(Boost_LIBRARY_DIRS $ENV{BOOST_LIBRARYDIR})
   endif()
   set(Boost_ADDITIONAL_VERSIONS "1.46.1" "1.45" "1.44" "1.43" "1.42" "1.41" "1.40" "1.39" "1.38" "1.37.0" "1.37" "1.35.0" "1.34.1" "1.34.0" "1.34" "1.33.1" "1.33.0" "1.33" ${Boost_ADDITIONAL_VERSIONS})
-  find_package(Boost COMPONENTS regex filesystem system python thread iostreams date_time)
+  if(OPT_PYTHON AND NOT USE_PYBIND11_PYTHON_BINDINGS)
+    find_package(Boost COMPONENTS regex filesystem system python thread iostreams date_time)
+  else()
+    find_package(Boost COMPONENTS regex filesystem system thread iostreams date_time)
+  endif()

```

Comment:
Why are these changes here? Seems unrelated to this PR

Reply:
Oh, I actually merged from `origin/production` (so that it effectively branches from that branch). So, this may be the result of that branch merge.